packages/react-native/template/android/app/src/main/java/com/helloworld/MainApplication.kt

```diff
@@ -15,11 +15,11 @@ class MainApplication : Application(), ReactApplication {
 
   override val reactNativeHost: ReactNativeHost =
       object : DefaultReactNativeHost(this) {
```

Comment:
The mix usages of 4 space tab and  2 space tab bothering me a lot 😞  

Can I make PR? ( asking cause this doesn't resolve or implement anything )

Reply:
This is autoformatted by ktfmt, so even if you send a PR that attempts to fix this, it will be reformatted back to how it is nwo